.github/workflows/lint-js.yml

```
@@ -43,14 +42,24 @@ jobs:
-        id: jsfiles
-        run: |
-          echo "JS_FILES<<EOF" >> $GITHUB_ENV
-          git diff --name-only ${{ github.event.before }} ${{ github.sha }} | grep '\.js$' || echo ""
-          echo "EOF" >> $GITHUB_ENV
-
-      - name: 'Run ESLint'
-        if: env.JS_FILES != ''
-        run: npm run lint:js -- ${{ env.JS_FILES }}
+      - name: Get changed files
+        id: changed-files
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)